rest-service/manager_rest/rest/rest_utils.py

```diff
@@ -703,8 +710,12 @@ def decrease_deployment_counts_in_graph(self, source):
                 )
                 if target.sub_services_count:
                     target.sub_services_count -= total_services
+                    if not target.sub_services_count:
+                        target.sub_services_count = None
                 if target.sub_environments_count:
                     target.sub_environments_count -= total_environments
+                    if not target.sub_environments_count:
```

Comment:
Same note as previous comment- this should maybe not be allowing null values.

Reply:
Sure